doc/freescout.md

```diff
@@ -0,0 +1,99 @@
+# Freescout
+
+[Freescout](https://freescout.net/) is used on top of [email addresses](./mail.md) for shared inboxes, in particular our own instance: <https://freescout.nixos.org/>.
+
+## Requests
+
+You can request a shared inbox for your team, or changes to an existing inbox by opening an issue in this repository and pinging @infinisil.
+
+For a new shared inbox, make sure to include:
+- Team name (will be used in email footer):
+- Email address: <email-handle>@nixos.org
+
+<details><summary>@infinisil perform these steps to set up a new shared inbox:</summary>
+
+- Generate a password for the email login (used to set up Freescout)
+- Create a PR to `mailing-lists.nix` for the new email address:
+  ```bash
+  nix run .#encrypt-email login <email-handle>
+  ```
+
+  ```nix
+    "<email-handle>@nixos.org" = {
+      loginAccount = {
+        encryptedHashedPassword = ../../secrets/<email-handle>-email-login.umbriel;
+        storeEmail = true;
+      };
+    };
+  ```
+- Create a new freescout mailbox accordingly:
+  - Set the mailbox name to the team name
+  - Enable access for people that already have a Freescout account
+  - In Edit Mailbox, set the email signature to
+    ```
+    --
+    {%user.fullName%}
+    {%mailbox.name%}
+    ```
+  - In Connection Settings > Fetching Emails:
+    - Server: `umbriel.nixos.org`
+    - Port: 993
+    - Username: <email-handle>@nixos.org
+    - Password: *************
+    - Encryption: TLS (+StartTLS)
+    - IMAP Folders: INBOX, Junk
+      - Note: Legitimate mail has ended in the junk folder before
+    - IMAP Folder To Save Outgoing Replies: Sent
+  - In Permissions, check "Hide from Assign list" for "Silvan (admin) Mosberger"
+- Invite everybody that doesn't have an account yet to freescout with access to the new mailbox
+  - Look up emails in the maintainer list
+
+</details>
+
+## Freescout quick start
+
+After you requested an account, you will receive a personal email titled "Welcome to Official Nix!".
```

Comment:
That sounds like a super confusing name.

Reply:
Yeah agreed, suggestions welcome, perhaps "Nix Freescout" or "Shared Nix Inboxes" or "Nix Email" :thinking:

Reply:
"NixOS Shared Mailbox Service" is what comes to mind. Not sure why "Nix" over "NixOS", when the latter is the foundation and domain name.

Reply:
Done